execution/record.go

```diff
@@ -228,6 +234,34 @@ func (r *Record) Hash() (uint64, error) {
 	return hashstructure.Hash(append(values, fields...), nil)
 }
 
+// GetRandomRecordID can be used to get a new random RecordID.
+func NewRecordID(id string) *RecordID {
```

Comment:
Is this exactly the same as `NewId` function from the beginning of this file? :P